### PR TITLE
Refactor/access vrender core

### PR DESCRIPTION
**Pull request by neuqzxy**
[[中文版模板 / Chinese template](https://github.com/VisActor/vgrammar/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Chore
- [ ] Release
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vgrammar-core/package.json [hunk 1]
```diff
@@ -39,8 +39,9 @@
     "@visactor/vgrammar-coordinate": "workspace:0.7.6",
     "@visactor/vgrammar-util": "workspace:0.7.6",
     "@visactor/vscale": "~0.16.0",
-    "@visactor/vrender": "~0.15.2",
-    "@visactor/vrender-components": "~0.15.2",
+    "@visactor/vrender-core": "~0.16.0",
+    "@visactor/vrender-kits": "~0.16.0",
```
Comment (xile611) [inline packages/vgrammar-core/package.json:43]: vrender-kits 现在用到了吗

File: packages/vgrammar-wordcloud/__tests__/cloud-layout.test.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import { CloudLayout } from '../src/cloud-layout';
+import { initBrowserEnv } from '@visactor/vgrammar-core';
```
Comment (xile611) [inline packages/vgrammar-wordcloud/__tests__/cloud-layout.test.ts:2]: 测试不了了吗？

File: packages/vgrammar-core/package.json (hunk 1 above)
Comment (neuqzxy) [reply packages/vgrammar-core/package.json:43]: 用了，环境相关代码都在kits里

File: packages/vgrammar-wordcloud/__tests__/cloud-layout.test.ts (hunk 2 above)
Comment (neuqzxy) [reply packages/vgrammar-wordcloud/__tests__/cloud-layout.test.ts:2]: > 测试不了了吗？

需要手动设置当前环境

**Pull request merged**